railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -66,13 +67,21 @@ end
 
 <%- if depends_on_system_test? -%>
 group :test do
+  # These gems are needed for System Testing: https://guides.rubyonrails.org/testing.html#system-testing
+
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 3.26'
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
 end
+<%- end -%>
+
+<% if depend_on_bootsnap? -%>
+# Speed up boot time by caching dependencies; configured in config/boot.rb
```

Comment:
Could we add the link here to bootsnap?

Reply:
I didn't include one originally because bootsnap is meant to "just work", but I'm not sure that's a great rationale. Will add.